MPF.Core/Modules/Redumper/Parameters.cs

```diff
@@ -1134,6 +1134,9 @@ protected override void SetDefaultParameters(string? drivePath, string filename,
 
             this[FlagStrings.Retries] = true;
             RetriesValue = options.RedumperRereadCount;
+
+            this[FlagStrings.PlextorLeadinRetries] = true;
+            PlextorLeadinRetriesValue = options.RedumperLeadinRetryCount;
```

Comment:
Did you intend on having this option on all the time? It seems like if this is set to 0, it should be disabled?

Reply:
The default value of 4 *should* be identical to omitting the flag, as redumper has 4 retries by default. Of course this is subject to change in redumper code.
I copied the reread tries option for this one.

Reply:
Thank you. The reason that I think this would be better to have as an enable-only option is that it does clutter up the resulting commandline unnecessarily for most users.